scss/civicrm/common/_backgrounds.scss

```diff
@@ -21,7 +21,7 @@
     .crm-form-block {
       @extend %block-with-shadow;
       border-radius: 0;
-      margin: 0;
+      margin-bottom: 15px;
```

Comment:
Why do we add exactly `15px`? Is there a variable for that?

If you have a look at `scss/bootstrap/overrides/_variables.scss` and search for `15px` you will see this value is actually used in a lot of places. It seems to me like this value is pretty universal and it should be defined in a separate variable like `$standard-vertical-grid-gap` and then using it everywhere when you use `15px` because it is a generic grid design decision.